en_US/advanced/acl-redis.md

```diff
@@ -145,5 +145,10 @@ You can adjust the ACL query command according to business requirement. However,
 
 1. Topic is used as key and access is used as value in hash
 
-::: tip 
-The above data structures   need to be used strictly for Redis ACL rules.All rules added in Redis ACL are `allow` rules, which can be used with `acl_nomatch = deny` in` etc / emqx.conf`. :::
\ No newline at end of file
+::: tip
+The above data structures need to be used strictly for Redis ACL rules.
+All rules added in Redis ACL are `allow` rules. It means a force whitelist.
+The ACL check would be failed when ACL rule list is not empty for specific topic.
+And the ACL chain will be terminated.
+It's recommended to set Redis ACL be the last one of ACL chain.
```

Comment:
```suggestion
ACL rules from redis are all **allow** rules. i.e. a whitelist.

When the rules list of a given topic is empty, EMQ X continues to check the next auth/ACL plugin.
Otherwise the check returns immediately without proceeding to the next auth/ACL plugins.

When more than one auth/ACL plugins are in use, it is recommended to position MongoDB ACL after other auth/ACL plugins in the enabled plugins list.
```